<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_register1Class</class>
 <widget class="QMainWindow" name="_register1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>xiuxian</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">*{
	background:rgb(255, 255, 255);
	font-size:15px;
	font-family:Century Gothic,sans-serif;
}
QFrame{
	border:sold 10px rgba(0,0,0);
	
	background-image: url(:/xiuxian/images/backgroundlogin.jpg);
}
QLineEdit{
	color: #8d98a1;
	background-color:#405361;
	font-size:18px;
	border-style:outset;
	border-radius:10px;
}

QCheckBox{
	background:rgba(85,170,225,0);
	color:white;
}

QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-31</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="userEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdEdit">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>170</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background:#ced1d8;
	border-style:outset;
	border-radius:10px;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/xiuxian/images/min.png);
	border:none;
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/xiuxian/images/close.png);
	border:none;
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="cfpwdEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>confirm password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minBtn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	image: url(:/xiuxian/images/arrow-left-circle.svg);
	border:none;
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
